eslint-local-rules.js

```diff
@@ -1,7 +1,9 @@
 var noBareTemplates = require('./packages/@okta/eslint-plugin-okta-ui/lib/rules/no-bare-templates');
 var noMissingKeys = require('./packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-keys');
+var noMissingAPIKeys = require('./packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys');
 
 module.exports = {
   'no-bare-templates': noBareTemplates,
   'no-missing-keys': noMissingKeys,
+  'no-missing-api-keys': noMissingAPIKeys
```

Comment:
Consider adding a [configuration](https://eslint.org/docs/developer-guide/working-with-plugins#configs-in-plugins) for all i18n related rules.

Reply:
Will do that as a follow up https://oktainc.atlassian.net/browse/OKTA-389367